### DISCUSSION TITLE
Unity for naming of sub classes

### DISCUSSION THREAD

**Discussion by KC45jp** [PygmalionOfCyprus/cmo-db-requests#1604] [Suggestions]
I notice that today but in DB, we have two types of way showing the sub-class.

One of the type is like DDG 19 Tattnall [Charles F. Adams] (Ship ID 530), one without word "class" in the brackets and
and Another one is like DDG Mahan [Farragut class] (SHIP ID 2000), one with "class" in the brackets.

I know I am talking about picky thing (I'm sorry if you don't like to talk this kind of things), but I thought it might be better to have some rules of naming if there are a lot of cases like this.

**Comment by PygmalionOfCyprus** [comment]
Oh, trust me, I've been tearing my hair out over this for a while. It's not easy to fix: if it was as simple as just appending "Class" to the end of every bracketed statement that'd be one thing, but the brackets serve multiple purposes...

We're actually considering adding a dedicated "Class" field for this sort of thing, but for now there are larger fish to fry.

**Reply by KC45jp** [reply]
I see. Thank you!
